README.md

```diff
@@ -62,9 +81,9 @@ plugin.configure({ transpileTemplates: false });
 ```
 
 ### transpileTemplates
-`true`  
```

Comment:
Can you re-add the whitespace from these two lines as they're needed for line breakage

Reply:
Grr, my editor removed the end-of-line whitespace automatically. Never seen end-of-line whitespace actually matter before.